.github/workflows/publish-release-zetacore.yml

```diff
@@ -0,0 +1,128 @@
+name: Publish ZetaCore Release
```

Comment:
if there is a duplicate between these 2 workflows, would it make sense to extract some common parts, like having more generic publish-release workflow and something like:

```
name: Publish ZetaClient Release
on:
  workflow_dispatch:
    inputs:
      version:
        description: 'Version for ZetaClient Release'
        required: true
      skip_checks:
        type: boolean
        default: false
      skip_release:
        type: boolean
        default: false

jobs:
  publish:
    uses: ./.github/workflows/publish-release.yml
    with:
      component: zetaclient
      version: ${{ inputs.version }}
      skip_checks: ${{ inputs.skip_checks }}
      skip_release: ${{ inputs.skip_release }}
    secrets: inherit

```

and similar to zetacore